### ISSUE TITLE
Arch & Background Section: 1,2 - Editorial

### ISSUE THREAD

**Issue by gorryfair** [taps-api/drafts#73]
/even when the functions are analogous./
- I am not sure what this is hinting at, maybe we could use simpler language? 
---
/This variety can lead to confusion when trying to define the fundamental commonalities between APIs and distill the nuanced differences./

I understood second read, but I did find it unnecessarily hard to parse, and simpler language would have been easier. 

I found the Background section a over-excited about how e approached the problem, rather than what we had to offer the reader. What I am trying to say is: I think it would help to say that a higher-level richer API is useful and why. One way is to move the text directly below.

This I think could usefully appear earlier in this document:
/The goal of the Transport Services architecture is to redefine the interface between applications and transports in a way that allows the transport layer to evolve and improve without fundamentally changing the contract with the application. This requires a careful consideration of how to expose the capabilities of protocols./
- I'd prefer this text to also assert that the higher layer interface /can enable the network to bring more context allowing the stack to making decisions about which transport protocols, options and paths are used to try to connect to a specific endpoint. /

**Comment by tfpauly** [comment]
Fixed the first two points about text in the introduction.

For the point about background, and the goal statement, we do have some explanation of the goal in the Introduction. You are correct that the background is very much about the process, and that was the intention, at least for this first round of the draft to make sure the WG knew where this stood in comparison to the rest of the work.

My intent was:

1. Introduction covers the fact that we need a common architecture for transport interfaces, and that this interface is meant to allow transport protocol evolution and development. Mentions the relationship with two other documents.
2. Background goes back to explain where this work came from.
3. Design principles covers the heart of the approach, taking the basic idea stated in the introduction and adding flesh to it

**Comment by gorryfair** [comment]
I think the current text is good enough to go forward.
